core/kernel/transfer_list.c

```diff
@@ -0,0 +1,542 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2023, Linaro Limited
+ */
+
```

Comment:
Could there be a link to the spec as an inline comment? It's currently only found in the commit message.

Reply:
I will add the link to the spec's Github project https://github.com/FirmwareHandoff/firmware_handoff
Then we can have a consitent link and avoid to update the link each time to different spec release.